npm-shrinkwrap.json

```diff
@@ -2555,23 +2555,10 @@
       "from": "form-data@>=2.1.1 <2.2.0",
       "resolved": "https://registry.npmjs.org/form-data/-/form-data-2.1.2.tgz",
       "dependencies": {
-        "async": {
-          "version": "0.9.2",
-          "from": "async@>=0.9.0 <0.10.0",
-          "resolved": "https://registry.npmjs.org/async/-/async-0.9.2.tgz",
-          "dev": true
-        },
-        "mime-db": {
```

Comment:
Looks like it got removed from the newest versions of `form-data` (https://github.com/form-data/form-data/blob/v2.1.2/package.json#L39), but npm didn't remove it from our shrinkwrap file for some reason.